### PR TITLE
log cleanup for snapshots

### PR DESCRIPTION
**Pull request by sivanbar**

### CODE REVIEW

File: content/manager/snapshots.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ Common use cases for snapshots are:
 * Migrating to a newer Cloudify Manager version, by creating a snapshot on the old manager and restoring it on the new one [see upgrade docs]({{< relref "installation/upgrade_4-0-0.md" >}}).  
   In this case, if you have any **installed** deployments, you also need to execute the [install_new_agents]({{< relref "workflows/built-in-workflows.md#the-install-new-agents-workflow" >}}) workflow on the new Cloudify Manager so that all hosts agents are updated and connected to RabbitMQ on the new Cloudify Manager.
 
+{{% gsNote title="Note" %}}
+To optimize snapshot size, cleanup old logs and events before taking a snapshot. For more information see [log cleanup]({{< relref "manager/service-logs.md#log-cleanup" >}}).
```
Comment (bmansheim) [inline content/manager/snapshots.md]: Instead of > For more information see
Change to > For more information, go to:

Comment (bmansheim) [reply content/manager/snapshots.md]: (I made a mistake in the review process so I had to add this comment)

File: content/manager/snapshots.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Common use cases for snapshots are:
   In this case, if you have any **installed** deployments, you also need to execute the [install_new_agents]({{< relref "workflows/built-in-workflows.md#the-install-new-agents-workflow" >}}) workflow on the new Cloudify Manager so that all hosts agents are updated and connected to RabbitMQ on the new Cloudify Manager.
 
 {{% gsNote title="Note" %}}
-To optimize snapshot size, cleanup old logs and events before taking a snapshot. For more information see [log cleanup]({{< relref "manager/service-logs.md#log-cleanup" >}}).
+To optimize snapshot size, cleanup old logs and events before taking a snapshot. For more information go to [log cleanup]({{< relref "manager/service-logs.md#log-cleanup" >}}).
```
Comment (bmansheim) [inline content/manager/snapshots.md]: Sorry to be annoying, but what exactly do we want customers to do in order to cleanup old logs and events? It seems from the target page that they can go to the various locations and delete logs, and configure log rotation. Is that correct?

**Comment by bmansheim** [comment]
FYI: This PR is still open with my question.

File: content/manager/snapshots.md (hunk 2 above)
Comment (sivanbar) [reply content/manager/snapshots.md]: It should point to the log cleanup section so they can delete logs and events

**Pull request merged**